src/content/docs/installation/filesystem.md

```diff
@@ -1,3 +1,114 @@
+- Resistant to fragmentation due to its extent-based nature and delayed allocation strategy.
+### Cons
+- Cannot be shrunk.
+
+### Userspace utility
+The package containing userspace tools to manage xfs filesystems is 'xfsprogs'.
+
+### Recommendation:
+XFS is the recommended filesystem for users who do not need advanced features and simply want a fast and reliable filesystem.
+
+
+## BTRFS
+BTRFS is a modern copy-on-write(COW) filesystem created in 2007 and declared stable in the linux kernel in 2013. It is widely supported and is mainly known for its advanced featureset.
+### Pros
+- Transparent compression. BTRFS supports transparently compressing files to allow for signifcant space savings with no user intervention. CachyOS ships with ZSTD compression by default
+- Snapshot functionality. BTRFS leverages its COW nature to allow for the creation of snapshots of subvolumes that take up very little actual space.
+- Subvolume functionality allowing for greater control over the filesystem.
+- Able to grow or shrink.
+- Very fast developement.
+### Cons
+- Sometimes requires defragmentation or balancing.
+- Worse on rotational drives due to aforementioned fragmentation.
+- 
+### Userspace utility
+Btrfs's userspace utility package is 'btrfs-progs'
+
+### Subvolume Layout
+CachyOS provides a subvolume layout out of the box to allow easy snapshot functionality.
+- Subvol @ = /
+- Subvol @home = /home
+- Subvol @root = /root
+- Subvol @srv = /srv
+- Subvol @cache = /var/cache
+- Subvol @tmp = /var/tmp
+- Subvol @log = /var/log
+
+### Recommendation:
+BTRFS is recommended for users who want snapshot/backup functionality and transparent compression.
+
+
+## Ext4
+Ext4 (fourth extended filesystem) is the most commonly used Linux filesystem. Ext4 was made stable in the linux kernel in 2008.
+### Pros
+- Very common allowing easy access to plenty of resources.
+- Reliable. Ext4 has a proven track record of being very reliable.
+- Able to grow or shrink.
+### Cons
+- Built on an old code base.
+- Lacks many of the advanced features other filesystems offer.
+
+### Userspace utilities
+The package to manage ext4 is 'e2fsprogs'
+
+### Recommendation:
+Ext4 is recommended for users who want the simplest and most commonly used filesystem.
+
+
+## ZFS
+ZFS is an advanced filesystem originally developed by Sun Microsystems in 2005. ZFS has many features however it licensed under CDDL which means it can not be included inside the linux kernel and requires a seperate module installed.
+### Pros
+- Pooled storage (zpool)
+- Snapshots using COW
+- Raid-Z support
+- ARC cache allows insanely fast read times on commonly accessed files.
+### Cons
+- Very complicated to use and understand due to features like zpool and ARC.
+- ARC requires a lot of ram to be effective.
+- Not included in the linux kernel therefore dependent on a third party kernel module (OpenZFS)
+
+### Required tools
+'zfs-dkms' or the precompiled module for your specific kernel is REQUIRED.
```

Comment:
I think we should highlight here that cachyos has each kernel its own zfs module